english/dance-randomly/dance-randomly.md

```diff
@@ -10,4 +10,6 @@ You catch yourself smiling, ever so brightly.
 
 And oh yeah... how could you have forgotten?... life... it's beautiful!
 
-It only makes sense that next, you...
\ No newline at end of file
+It only makes sense that next, you...
+
+[join the alliance of hip-shaking](../join alliance/dance-randomly.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.